ledger-core/v2/application/functest/create_wallet_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+// +build functest
+
+package functest
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// Creates wallets and check Reference format in response body.
+func TestCreateWallet(t *testing.T) {
+	for i := 0; i < 100; i++ {
```

Comment:
Shouldn't  we use for that "count" param in "go test"?

Reply:
I think, that `go test -count=100` is better to use for all tests in the package. To run them multiple times, for example in the nightly build.

Reply:
I don't think so. However, current implementation is wrong in other way. Loop should be inside of `t.Run`. `t.Parallel()` shouldn't be used in cases when we want to do operations in parallel in a single tests. Purpose of `t.Parallel()` is to tell that some group of tests can be run in parallel and there is no know interference between tests.

Reply:
done